src/composables/networks.ts

```diff
@@ -150,18 +150,23 @@ export function useNetworks() {
  * user changes the network.
  */
 export function createNetworkWatcher() {
-  ensureDefaultNetworksExists();
+  // Delay the networks fill in to allow the protocol adapters to be created first
```

Comment:
Feel like a workaround rather than actual solution. And the source of the failing test (IMO) is behind the source of this changes.

Reply:
I created separate task for this: https://github.com/superhero-com/superhero-wallet/issues/2595